aim/web/ui_v2/src/services/models/metrics/metricsAppModel.ts

```diff
@@ -97,12 +99,22 @@ function getConfig() {
 
 function initialize() {
   model.init();
+  const configData: IMetricAppConfig = getConfig();
+  const groupingData: IMetricAppConfig['grouping'] =
+    getStateFromUrl('grouping');
+  const chartData: IMetricAppConfig['chart'] = getStateFromUrl('chart');
+  if (groupingData) {
```

Comment:
@arsengit it's better to deep merge the initial config with the one recovered from URL.
e.g.
```js
_.merge({}, getConfig(), {
    grouping, // recovered from URL
    chart, // recovered from URL
})
```

Reply:
@roubkar fixed